docs/setup/install/brew.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[brew]]
+=== Install {kib} on macOS with Homebrew
+
+Elastic publishes Homebrew formulae so you can install {kib} with the https://brew.sh/[Homebrew] package manager.
+
+To install with Homebrew, you first need to tap the Elastic Homebrew repository:
+
+[source,sh]
+-------------------------
+brew tap elastic/tap
+-------------------------
+
+Once you've tapped the Elastic Homebrew repo, you can use `brew install` to
+install {kib}:
+
+[source,sh]
+-------------------------
+brew install elastic/tap/kibana
+-------------------------
+
+This installs the most recently released default distribution of {kib}.
+Append the version number to install a previous version, for example
+`elastic/tap/kibana@6.8`. To install the OSS distribution,
```

Comment:
Remove "Append the version number to install a previous version, for example
`elastic/tap/kibana@6.8`."